### PR TITLE
CASMUSER-3190

### PR DESCRIPTION
**Pull request by darrenn-jackson**
#### Summary and Scope

- Implements CASMUSER-3190

##### Issue Type

**Docs Pull Request**

- Replaces link to Getting Started Guide with a link to its replacement. The GSG doesn't exist in the next CSM recipe release.
- Breaks up `UAN_Ansible_Roles.md` file into separate files, one for each role.
- updates both ditamaps and the `hugo_toc.json` file for the refactoring
- replaced links to the deleted file with links to the specific role.

#### Prerequisites

- [x] I have included documentation in my PR (or it is not required)
- [x] I tested this via local builds on my WSL2/Ubuntu instance and uploading to the test ITG portal. No errors from DITA-OT. Content on ITG test portal looks good.
 
#### Idempotency
 
N/A
 
#### Risks and Mitigations
 
Slight risk that I updated the `hugo_toc.json` file incorrectly since I can't test that build process.

### CODE REVIEW

File: docs/portal/developer-portal/operations/Basic_UAN_Configuration.md
Comment (alanm-hpe) [inline docs/portal/developer-portal/operations/Basic_UAN_Configuration.md:19]: Should these ansible readme markdowns be linked also?

Comment (darrenn-jackson) [reply docs/portal/developer-portal/operations/Basic_UAN_Configuration.md]: Yes they should. Thanks for catching that. I added links in my latest commit.

**Review by alanm-hpe — APPROVED** [review]

**Comment by darrenn-jackson** [comment]
waiting for @keopp-hpe 's approval before merging so that he can assess any impact to the Hugo-generated docs.

**Pull request merged**
